### PR TITLE
Update da_DK translations.php

### PR DESCRIPTION
**Pull request by twikedk**
Before to submit your pull-request:

- Be sure that the unit tests pass
- If you create a new feature, test your code, and do not introduce new bugs
- Avoid code duplication
- Remember, small pull-requests are easier to review and can be merged quickly
- 1 pull-request == 1 feature/improvement
- Nonessential features should be implemented as plugin

### CODE REVIEW

**Comment by twikedk** [comment]
A total rework of the danish translation with aim on KIS with reference to Dansk-gruppen suggestion

File: app/Locale/da_DK/translations.php [hunk 1]
```diff
@@ -1,1378 +1,1378 @@
...
-    // 'This feature enables the iCal feed, RSS feed and the public board view.' => '',
-    // 'Stop the timer of all subtasks when moving a task to another column' => '',
-    // 'Subtask Title' => '',
-    // 'Add a subtask and activate the timer when moving a task to another column' => '',
-    // 'days' => '',
-    // 'minutes' => '',
-    // 'seconds' => '',
-    // 'Assign automatically a color when preset start date is reached' => '',
-    // 'Move the task to another column once a predefined start date is reached' => '',
+	'number.decimals_separato
```
Comment (fguillot) [inline app/Locale/da_DK/translations.php]: The placeholder `%s` is missing here.

Comment (twikedk) [reply app/Locale/da_DK/translations.php]: Yes I found it and some other minors, should I make a new PR then??

Comment (fguillot) [reply app/Locale/da_DK/translations.php]: You could update this PR until the tests are green or make a new one. It's up to you. Thanks

**Comment by fguillot** [comment]
Closing this PR because #3764 has been merged.

**Pull request closed without merging**